_specifications/lsp/3.18/language/signatureHelp.md

```diff
@@ -217,15 +227,24 @@ export interface SignatureHelp {
 	activeSignature?: uinteger;
 
 	/**
-	 * The active parameter of the active signature. If omitted or the value
-	 * lies outside the range of `signatures[activeSignature].parameters`
-	 * defaults to 0 if the active signature has parameters. If
-	 * the active signature has no parameters it is ignored.
+	 * The active parameter of the active signature.
+	 * 
+	 * If `null`, no parameter of the signature is active (for example a named
+	 * argument that does not match any declared parameters). This is only valid
+	 * since 3.18.0 and if the client specifies the client capability
+	 * `textDocument.signatureHelp.noActiveParameterSupport === true`
+	 * 
+	 * If omitted or the value lies outside the range of
+	 * `signatures[activeSignature].parameters` defaults to 0 if the active
+	 * signature has parameters.
+	 * 
+	 * If the active signature has no parameters it is ignored.
+	 * 
 	 * In future version of the protocol this property might become
-	 * mandatory to better express the active parameter if the
-	 * active signature does have any.
+	 * mandatory (but still nullable) to better express the active parameter if
+	 * the active signature does have any.
 	 */
```

Comment:
I usually do the following:

@since 3.18.0: support for `null`

Any reason why this shouldn't work

Reply:
@dbaeumer is it intended that this would happen in the case where there isn't another `@since`?

<img width="730" alt="Screenshot 2023-03-06 at 16 35 34" src="https://user-images.githubusercontent.com/1078012/223173233-2b2c34ba-62c4-4fb2-9e06-6770b36d01a5.png">

When there's another, then it looks like this:

<img width="1004" alt="Screenshot 2023-03-06 at 16 37 02" src="https://user-images.githubusercontent.com/1078012/223173672-10426b4e-857e-43c2-953b-8664cf1d8974.png">

There's some similar instances in the existing meta model:

https://github.com/microsoft/vscode-languageserver-node/blob/4d7d8f4229a539f36bc28c634470539bcda8b362/protocol/metaModel.json#L4680

I don't know if `"since"` is expected to be parseable as a version number?